sites/public/src/components/listing/GetApplication.tsx

```diff
@@ -44,6 +47,8 @@ export interface ApplicationsProps {
   listingStatus?: string
   /** Whether or not to block submission of test application */
   preview?: boolean
+  /** Whether or not to direct user to sign in page before applying*/
```

Comment:
Looks like this isn't used as a heads up

Reply:
Good catch, will delete